src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -219,7 +219,7 @@ private function getContainerDefinitionData(Definition $definition, bool $omitTa
                 if ($factory[0] instanceof Reference) {
                     $data['factory_service'] = (string) $factory[0];
                 } elseif ($factory[0] instanceof Definition) {
-                    throw new \InvalidArgumentException('Factory is not describable.');
+                    $data['factory_class'] = $factory[0]->getClass() ?? 'not configured';
```

Comment:
this might be better describe using `factory_service: inline service ($class)` instead, making it clear that this is not doing a static call to the method (which is what happens for other cases using `factory_class`)

Reply:
Should I submit another PR?

Reply:
Woops just saw https://github.com/symfony/symfony/pull/47469